LayoutTests/http/tests/paymentrequest/ApplePayModifier-disbursementPaymentRequest.https.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Tests for providing `disbursementPaymentRequest` as part of `ApplePayModifier`.</title>
+<script src="/js-test-resources/ui-helper.js"></script>
+<script src="/resources/payment-request.js"></script>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+setup({ explicit_done: true, explicit_timeout: true });
+
+test((test) => {
+    const method = validPaymentMethod();
+    method.data.features = ["paymentRequestDisbursements"];
+    new PaymentRequest([method], validPaymentDetails());
+}, "Should have a feature for `disbursementPaymentRequest`.");
+
+let lineItems = {
+    total: {
+        "label": "Chime",
+        "amount": "15.00"
+    },
+    ifo: {
+        "label": "Instant Transfer Fee",
+        "amount": "0.15",
+        "disbursementLineItemType": "instantFundsOutFee"
+    },
+    disbursement: {
+        "label": "Total Amount",
+        "amount": "14.85",
+        "disbursementLineItemType": "disbursement"
+    }
+}
+
+let validLineItems = [ lineItems.total, lineItems.ifo, lineItems.disbursement ]
+
+function modifiersWithData(data, lineItems) {
+    data.additionalLineItems = lineItems
+    return [ { supportedMethods: 'https://apple.com/apple-pay', data } ];
+}
+
+function testValidDisbursementPaymentRequest(description, {initialDisbursementPaymentRequest, expectedDisbursementPaymentRequest}) {
+    user_activation_test(async (test) => {
+        let detailsInit = {
+            "total": {
+                "label": "Disbursement",
+                "amount": {
+                    "value": "14.85",
+                    "currency": "USD"
+                }
+            }
+        }
+        if (initialDisbursementPaymentRequest !== undefined) {
+            detailsInit.modifiers = modifiersWithData({ disbursementPaymentRequest: initialDisbursementPaymentRequest}, validLineItems);
+        }
+
+        let request = new PaymentRequest([validPaymentMethod()], detailsInit);
+        request.addEventListener("merchantvalidation", (event) => {
+            event.complete({});
+        });
+
+        request.addEventListener("shippingaddresschange", (event) => {
+            event.updateWith({ } );
+            internals.mockPaymentCoordinator.changePaymentMethod({ type: "debit" });
+        });
+
+        request.addEventListener("paymentmethodchange", (event) => {
+            let detailsUpdate = { };
+            event.updateWith({  });
```

Comment:
`detailsUpdate` is defined but never used. 

```suggestion
            event.updateWith({  });
```